readthedocsext/theme/templates/projects/edit_base.html

```diff
@@ -17,6 +17,9 @@
         <a class="item {% block project_advanced_active %}{% endblock %}" href="{% url "projects_advanced" project.slug %}">
           {% trans "Advanced settings" %}
         </a>
+        <a class="item {% block project_addons_active %}{% endblock %}" href="{% url "projects_addons" project.slug %}">
+            {% trans "Addons" %} <i class="ui small grey fa-duotone fa-rocket"></i>
```

Comment:
Instead of the icon here, I would try using a label -- I've wanted to add a marker for new/beta features this way. Menu elements will show the label floating to the right, you can see an example here:

https://fomantic-ui.com/collections/menu.html#vertical-menu

Untested, but:

```suggestion
            {% trans "Addons" %}
            <div class="ui basic primary label">
              {% trans "Beta" %}
            </div>
```

Reply:
Beautiful!


![Screenshot_2023-09-14_12-53-10](https://github.com/readthedocs/ext-theme/assets/244656/5ffa7dc7-ef49-49bc-aa0b-f0fb1c8530b3)


This is exactly what I wanted but didn't know how to do it. I originally tested 🆕 but that icon was not available so I put just a random icon to denote that _I wanted something to differentiate_ this entry.

Reply:
Perfect, that looks great. I'll want to reuse this pattern for new admin tabs and new features too, so glad we have a reason to try it out.